### PR TITLE
Form new trip

### PR DESCRIPTION
**Pull request by HannaSophia-92**

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -19,4 +19,9 @@ export const parameters = {
         <Story />
       </>
     ),
+    (Story) => (
+      <MemoryRouter>
+        <Story />
+      </MemoryRouter>
+    ),
```
Comment (mbosselmann) [inline .storybook/preview.js:26]: 👍

File: src/components/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { NavLink } from 'react-router-dom';
+import ScreenReaderOnly from './ScreenReaderOnly';
+import { HiHome } from 'react-icons/hi';
```
Comment (mbosselmann) [inline src/components/Navigation.js:3]: Same here for your icon names as mentioned above. Check also your other files. :-)

**Review by mbosselmann — COMMENTED** [review]
Good job! 🚀 Keep it up! 👍 But I have some remarks you should consider before merging. :)

File: src/components/FutureTripCard.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+import { FiTrash } from 'react-icons/fi';
```
[the remark this answers is not in the document]
Comment (HannaSophia-92) [reply src/components/FutureTripCard.js:2]: Great idea, thank you Mareike :)

File: src/components/FutureTripCard.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  justify-content: center;
+  margin: 10px;
+`;
+
+const Delete = styled.div`
+  position: absolute;
+  bottom: 10px;
+  right: 25px;
+
+  &&:hover {
```
[the remark this answers is not in the document]
Comment (HannaSophia-92) [reply src/components/FutureTripCard.js:39]: Good point :) It also worked out perfectly with two `&&`. I´m going to change it though :)

**Pull request merged**